app/src/main/java/com/celzero/bravedns/util/OrbotHelper.kt

```diff
@@ -243,7 +243,7 @@ class OrbotHelper(private val context: Context, private val persistentState: Per
             notificationManager.notify(ORBOT_SERVICE_ID, createNotification().build())
         }
         selectedProxyType = AppMode.ProxyType.NONE.name
-        appMode.removeProxy(AppMode.ProxyType.NONE, AppMode.ProxyProvider.ORBOT)
+        appMode.removeProxy(AppMode.ProxyType.NONE, AppMode.ProxyProvider.NONE)
```

Comment:
This is used in multiple places. I think, at this point, it might be prudent to simply expose `appMode#removeAllProxies()` to other objs?

Reply:
done.